docs/04-api-reference/front-commerce-core/ServerEvent.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: "ServerEvent"
+sidebar_position: 8
+description:
+  "The ServerEvent class define ServerEvent format used in Front-Commerce to be
+  published to the server event worker."
+---
+
+<p>{frontMatter.description}</p>
+
+## Usage:
+
+```ts
+import { ServerEvent } from "@front-commerce/core/events";
+
+// We define a static event Name for our event
+const MyEventName = "MyEvent";
+
+// Definition of the payload
+type MyEventPayload = {
+  sku: string;
+  price: boolean;
+};
```

Comment:
**thought:** I think the type's name is self-explanatory enough.

**suggestion:** Remove the comment.

```suggestion
type MyEventPayload = {
  sku: string;
  price: boolean;
};
```

Reply:
Fixed with your suggestion